frontend/src/metabase/search/components/SearchUserPicker/SearchUserPicker.styled.tsx

```diff
@@ -8,3 +11,42 @@ export const SearchUserPickerContainer = styled(Stack)`
 export const SearchUserPickerContent = styled(Stack)`
   overflow-y: auto;
 `;
+
+export const SearchUserSelectBox = styled(Group)`
+  border: ${({ theme }) => theme.colors.border[0]} 1px solid;
+  border-radius: ${({ theme }) => theme.radius.md};
+`;
+
+export const SelectedUserButton = styled(Button)<
```

Comment:
We don't seem to have a `light` variant of the `Button` component right now. This is the closest I can come up with but maybe it would be better to add that variant to the `theme.ts` file

Reply:
I recommend talking to designers first before doing this